sonar-ws/src/main/java/org/sonarqube/ws/client/HttpRequestFactory.java

```diff
@@ -136,8 +136,9 @@ public String execute(WsRequest wsRequest) {
 
   public <T extends Message> T execute(WsRequest wsRequest, Parser<T> protobufParser) {
     HttpRequest httpRequest = wsRequestToHttpRequest(wsRequest);
+    String response = execute(httpRequest);
     try {
-      return protobufParser.parseFrom(httpRequest.bytes());
+      return protobufParser.parseFrom(response.getBytes());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of "getBytes" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1943)